backends-common/elasticsearch/pom.xml

```diff
@@ -82,7 +82,7 @@
         <dependency>
             <groupId>org.elasticsearch.client</groupId>
             <artifactId>elasticsearch-rest-high-level-client</artifactId>
-            <version>6.7.2</version>
+            <version>6.3.2</version>
```

Comment:
you cannot use the 6.7 client with a 6.3 server?

Reply:
We tried before, cannot. The 6.7 client insert some params in its requests that 6.3 doesn't know about, breaks everything

Reply:
no, that is the issue raised by Nicolas Parque, es server 6.3 rejected a compulsory parameter send by es client version 6.7